### ISSUE TITLE
ECL verbosity

### ISSUE THREAD

**Issue by LorenzMeier** [PX4/ecl#225]
@priseborough It would be nice if ECL could be less verbose on boot. Changing from:

```
INFO  [lib__ecl] EKF IMU buffer length = 21
INFO  [lib__ecl] EKF observation buffer length = 16
INFO  [lib__ecl] EKF using pressure height - commencing alignment
INFO  [lib__ecl] EKF alignment complete
```
to:
```
INFO  [lib__ecl] EKF aligned (pressure height, IMU buf: 21, OBS buf: 16)
```

You can still keep any error messages. We're following the POSIX design pattern: No news is good news and components should offer as little positive notifications as possible.

On the flip side *every* error must be handled and reported. Taking this principle to the extreme does show errors much more easily as they stand out and people avoid reporting errors by not reporting success, which is a terrible pattern.

**Comment by nicolaerosia** [comment]
@LorenzMeier I believe this can be closed

**Comment by LorenzMeier** [comment]
Thanks! And I will work hard to get your PR's in this week after we've released 1.6.4
